recipe/install_nvcc.sh

```diff
@@ -64,11 +64,14 @@ export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include"
 export CPPFLAGS="\${CPPFLAGS} -I\${CUDA_HOME}/include"
 export CXXFLAGS="\${CXXFLAGS} -I\${CUDA_HOME}/include"
 
+CONDA_ENV_SYSROOT=${PREFIX}/${HOST}/sysroot
```

Comment:
Can you create this directory if it doesn't exist?

Reply:
The subsequent `mkdir -p "\${CONDA_ENV_SYSROOT}/lib"` handles this.